utils/getTrustedCertificates.ts

```diff
@@ -16,16 +16,27 @@ let _systemCertificates: (string | Buffer)[] | undefined;
 
```

Comment:
So if I specify one file in `docker.certificatePaths` does that mean I only get that one file and that's it? I would expect it to be _added_ to the default "hard-coded internal list of certificates"

Reply:
I think we can mitigate this by improving the default "it just works" behavior of the settings. It doesn't feel super related to this PR, so I filed a separate issue to discuss: https://github.com/Microsoft/vscode-docker/issues/633